src/loc/statement/StatementOfFactsForm.tsx

```diff
@@ -8,6 +8,8 @@ import { useCommonContext } from "../../common/CommonContext";
 import { FormValues, Language } from "./PathModel";
 
 import "./StatementOfFactsForm.css"
+import FileSelectorButton from "../../common/FileSelectorButton";
+import React from "react";
```

Comment:
Not sure this import is still needed.

Reply:
Correct. https://github.com/logion-network/logion-wallet/compare/1aa682cc039e90c541a64b542695a4a2368bc21d..89d5ef7b968f05b934e1e5c45ae8bc6f466bc6a3